src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -184,13 +189,26 @@ public class PublicationService : IPublicationService
 
                     if (publication.Live)
                     {
-                        await _publishingService.PublicationChanged(publication.Id);
+                        publication.Published = DateTime.UtcNow;
+                        await _context.SaveChangesAsync();
+
+                        await DeleteCachedTaxonomyBlobs();
+                        await _publicBlobCacheService.DeleteItem(new PublicationCacheKey(publication.Slug));
+                        // @MarkFix EES-3149 Need to handle superseded publication.json cache files here too?
```

Comment:
If you use `// TODO` rather than your own annotation it will benefit us all as we can see all TODO items in the Rider TODO tool window and we also get a prompt if we are altering code with a TODO still remaining.